frontend/src/app/products/components/forms/shopping-list-selector.tsx

```
@@ -58,46 +58,51 @@ export default function ShoppingListSelector({
                   variant="outline"
                   role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)